src/novel_applications/payment/private_payment.md

```diff
@@ -0,0 +1,167 @@
+
+- A modern, simple and clean UI
+- Ability to make transfers in a fast, cheap and privacy preserving way
+- Ability to create accounts with a simple onboarding flow (abstracted seed phrases/UX)
+- Built on top of Miden
+
+Taking inspiration from successful payment applications (e.i. Venmo or Revolut), we would prioritize simplicity and cleanliness for the interface, making the application approachable for users of any technical level. We can imagine a frontend that would aggregate all user assets and transfers, display a portfolio value and enable transfers:
+
+<p align="center">
+    <img src="../../assets/images/payment_application.png" width="40%" alt="Imagined payment app">
+</p>
+
+We could imagine additional features which could be added to this type of application:
+
+- Simple DeFi page, aggregating best sources of yield / lending / borrowing from different Miden protocols
+- Phone-to-phone tap payment
+- Creation of digital bank cards and integration with Apple / Google Pay
+- Creation of physical bank cards (Gnosis card, Crypto.com card, etc.)
+
+## Conclusion
+
+Miden represents a significant leap forward in blockchain-based payment systems, addressing key limitations of existing solutions. By combining privacy, programmability, and scalability, it opens the door to user-friendly applications that could revolutionize how we transact, as presented above.
+
+## Additional thoughts and questions
+
+#### A better UX on Miden: Account abstraction
+
+*Question: Why would a user have a better UX on Miden than on another blockchain?*
+
+Ethereum has set the standard for accounts in the VM-enabled blockchain world. There are two types of accounts in Ethereum:
+
+- Externally owned accounts (EOA): are created by generating a public/private ECDSA key pair. Does not hold code. Can initiate transactions.
+- Contract accounts (Smart contracts): are created by being deployed on the Ethereum blockchain by an EOA. Holds code which can be executed. Can't initiate transactions.
+
+We clearly understand here that Ethereum has made the choice to separate user accounts from executable code. What if we could merge both to enable programmable user accounts? Welcoming `Account abstraction`, which can be defined as follows:
+
+"Account abstraction is a method of setting up a blockchain network in which users' assets are stored exclusively in smart contracts, and not in external accounts (External Owned Accounts, EOAs). When using this approach, a crypto wallet turns into a unique smart contract that can be programmed for various purposes."
+
+[What is account abstraction and why is it important - Medium - Aleksander](https://medium.com/@alex-100/what-is-account-abstraction-and-why-is-it-important-9627a4ced4f3)
+
+Miden supports full account abstraction, enabling full programmability of user accounts defined as follows:
```

Comment:
I think you can skip this account picture and just link to the account page in our docs

https://docs.polygon.technology/miden/miden-base/architecture/accounts/